lib/jsonld-signatures.js

```diff
@@ -781,7 +781,7 @@ function _getDataToHash(input, options) {
     if(options.domain !== null && options.domain !== undefined) {
       toHash += '@' + options.domain;
     }
-  } else if(options.algorithm === 'LinkedDataSignature2015') {
+  } else if(options.algorithm === 'LinkedDataSignature2015' || options.algorithm === 'EcdsaKoblitzSignature2016') {
```

Comment:
That would be fine. There are already previous checks that the algorithm is supported, e.g.
```
  if(api.SUPPORTED_ALGORITHMS.indexOf(algorithm) === -1) {
    return callback(new Error(
      '[jsigs.sign] Unsupported algorithm "' + algorithm + '"; ' +
      'options.algorithm must be one of: ' +
      JSON.stringify(api.SUPPORTED_ALGORITHMS)));
  }
```